From 80fbc70365e553385d92a087a6f070f871fc0337 Mon Sep 17 00:00:00 2001
From: Shi Wenjun <b42754@freescale.com>
Date: Wed, 26 Sep 2012 14:29:33 +0800
Subject: [PATCH 064/602] ENGR00225910 remove redundant ril-daemon item for JB

There exist two ril-daemon start up items in init.rc.
Only need to keep one item.

Change-Id: Ia637732167eb62863bb69acece8ef4b6177e6c64
Signed-off-by: Shi Wenjun <b42754@freescale.com>
---
 imx6/etc/init.rc | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index 9a61a59..b2463e5 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -459,13 +459,6 @@ service netd /system/bin/netd
 service debuggerd /system/bin/debuggerd
     class main
 
-service ril-daemon /system/bin/rild
-    class main
-    socket rild stream 660 root radio
-    socket rild-debug stream 660 radio system
-    user root
-    group radio cache inet misc audio sdcard_rw log
-
 service surfaceflinger /system/bin/surfaceflinger
     class main
     user system
@@ -584,7 +577,7 @@ service ril-daemon /system/bin/rild -l /system/lib/libreference-ril.so
     socket rild-debug stream 660 radio system
     socket rild-ppp stream 660 radio system
     user root
-    group radio cache inet misc audio
+    group radio cache inet misc audio sdcard_rw log
 
 service pppd_gprs /etc/ppp/init.gprs-pppd
     class main
-- 
1.8.0

